infra/src/test/kotlin/fi/fta/geoviite/infra/linking/switches/SwitchLinkingServiceIT.kt

```diff
@@ -1381,73 +1380,6 @@ constructor(
         )
     }
 
-    @Test
-    @Disabled // work out whether this unrealistic situation actually needs to be supported
```

Comment:
Poistin tämän testin koska nykyinen topologialinkitys ei ole suunniteltu toimivan näin. Itse topologiakytkentä on puolestaan pelkkä aktiivisten noodien yhdistely, joten jos raiteella on joku vaihdenoodi olemassa (topologia tai ei) siitä ei helposti ole nähtävissä onko se jostain syystä väärä.

Uusi malli perustuu siihen että vaihteen linkityksessä sekä poistossa poistetaan linkki kaikilta olemassaolevilta raiteilta joten mitään "roska" topologiakytkentöjä ei pitäisi olla olemassa. Tällaiselle toiminnalle ei siis pitäisi olla tarvetta enää.